Sources/Commands/PackageCommands/Init.swift

```diff
@@ -63,7 +63,7 @@ extension SwiftPackageCommand {
             if testLibraryOptions.isEnabled(.xctest) {
                 supportedTestingLibraries.insert(.xctest)
             }
-            if testLibraryOptions.explicitlyEnableSwiftTestingLibrarySupport == true || testLibraryOptions.explicitlyEnableExperimentalSwiftTestingLibrarySupport == true {
+            if testLibraryOptions.isExplicitlyEnabled(.swiftTesting) == true {
```

Comment:
No need to use `==` operator here.

Reply:
It's an optional boolean, so yes need :)